src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/DiagnosticReportDateIssued.java

```diff
@@ -0,0 +1,70 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+import uk.nhs.digital.nhsconnect.lab.results.utils.TimestampService;
+
+/**
+ * Example DTM+ISR:202001280957:203'
+ */
+@EqualsAndHashCode(callSuper = false)
+@Getter
+@AllArgsConstructor
+@Builder
+public class DiagnosticReportDateIssued extends Segment {
+
+    private static final String KEY = "DTM";
+    private static final String QUALIFIER = "ISR";
+    private static final String KEY_QUALIFIER = KEY + "+" + QUALIFIER;
+    private static final String DATE_FORMAT = "203";
+
+    private final LocalDateTime dateIssued;
+
+    public static final DateTimeFormatter DATE_TIME_FORMATTER =
+        DateTimeFormatter.ofPattern("yyyMMddHHmm").withZone(TimestampService.UK_ZONE);
+
+
+    public static DiagnosticReportDateIssued fromString(final String edifactString) {
+        if (!edifactString.startsWith(DiagnosticReportDateIssued.KEY_QUALIFIER)) {
+            throw new IllegalArgumentException("Can't create " + DiagnosticReportDateIssued.class.getSimpleName()
+                + " from " + edifactString);
+        }
+
+        final String dateTime = Split.byColon(Split.byPlus(edifactString)[1])[1];
+        final LocalDateTime instant = LocalDateTime.parse(dateTime, DATE_TIME_FORMATTER);
+        return new DiagnosticReportDateIssued(instant);
+    }
+
+    @Override
+    public String getKey() {
+        return KEY;
+    }
+
+    @Override
+    public String getValue() {
+        return QUALIFIER
+            .concat(COLON_SEPARATOR)
+            .concat(DATE_TIME_FORMATTER.format(dateIssued))
+            .concat(COLON_SEPARATOR)
+            .concat(DATE_FORMAT);
```

Comment:
The `.concat()` method is really only there for streaming/aggregation operations. It's faster just to use `+`
```suggestion
        return QUALIFIER
            + COLON_SEPARATOR
            + DATE_TIME_FORMATTER.format(dateIssued)
            + COLON_SEPARATOR
            + DATE_FORMAT;
```

Reply:
Done in e8d301a